src/openforms/forms/api/serializers/form.py

```diff
@@ -245,16 +329,79 @@ def get_fields(self):
             fields["payment_backend"].choices = [
                 ("", "")
             ] + payment_register.get_choices()
+
+        # add fields for v2 backwards compatibility
+        if "registration_backends" in fields:
+            fields["registration_backend"] = fields[
```

Comment:
have you checked if this returns the original instance or a copy of it? I think it's the latter, but it would suck if this causes the validation to be relaxed in the new serializer :thinking:

Reply:
It isn't the same instance as `FormRegistrationBackendSerializer().fields['backend']`, but I'm not sure if that's enough. I'll rewrite this in a way I'm sure it's not the same instance.